app/frontend/package.json

```diff
@@ -45,6 +33,17 @@
     "xlsx": "^0.17.0"
   },
   "devDependencies": {
+    "@ckeditor/ckeditor5-basic-styles": "^29.1.0",
+    "@ckeditor/ckeditor5-dev-utils": "^25.4.0",
+    "@ckeditor/ckeditor5-dev-webpack-plugin": "^25.4.0",
+    "@ckeditor/ckeditor5-editor-classic": "^29.1.0",
+    "@ckeditor/ckeditor5-essentials": "^29.1.0",
+    "@ckeditor/ckeditor5-link": "^29.1.0",
+    "@ckeditor/ckeditor5-paragraph": "^29.1.0",
+    "@ckeditor/ckeditor5-source-editing": "^29.1.0",
+    "@ckeditor/ckeditor5-table": "^29.1.0",
+    "@ckeditor/ckeditor5-theme-lark": "^29.1.0",
+    "@ckeditor/ckeditor5-vue2": "^1.0.5",
```

Comment:
Are you sure these should be dev dependencies? The main way you determine if something is a dev dependency or a primary dependency is whether or not you are directly importing ore requiring the library in your own written code. If you're pulling it in yourself in your written code, it's a primary dependency; if it's something that's only used by an auxiliary tool and not directly needed outside of compile time for example, then it's a dev dependency (like Jest).

Reply:
ok. thanks for the explaination. i've moved these back to 'non-dev' dependencies